Makefile

```diff
@@ -1,3 +1,5 @@
+REGISTRY := github.com/open-component-model/signing-server
```

Comment:
Why did the registry change? Also, this is not actually an image registry?

Reply:
Yes but this registry is not valid. It needs to be ghcr.io if at all.